src/controllers/login.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response, NextFunction } from 'express'
+import passport from 'passport'
+import jwt from 'jsonwebtoken'
+import {
+  NotFoundError,
+  UnauthorizedError,
+  InternalServerError,
+  BadRequestError,
+} from '../helpers/apiError'
+import JobSeeker from '../entities/JobSeeker.postgres'
+
+// Auth Controllers for job seeker
+export const jobSeekerLocalLogin = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  passport.authenticate('local', (error, jobSeeker: JobSeeker, info) => {
```

Comment:
Since this is used for both `jobseeker` and `employer` login, the second parameter that you receive could be either of them, hence change the name and also the type:
`(error, user: JobSeeker | Employer)`